docs/sandbox-permissions.rst

```diff
@@ -109,10 +109,10 @@ D-Bus permissions needed.
 
 **Ownership**
 
-Applications are automatically granted access to their own namespace. Ownership
-beyond this is typically unnecessary, although there are a small
-number of exceptions, such as using `MPRIS to provide media controls
-<https://www.freedesktop.org/wiki/Specifications/mpris-spec/>`_.
+Applications are automatically granted access to their own namespace as well
```

Comment:
Mentioning portal and the bus itself will be good too. https://github.com/flatpak/flatpak/pull/5582/files#diff-c1a0818d4158cb0c802cf9432c8b2b7af7c810bdfc1d1fece2172b73f74bd1fbR729-R731

Reply:
This is specifically in the *ownership* section.

Reply:
we should clarify what the automatically granted MPRIS bus name is.

Reply:
> This is specifically in the ownership section.

Ah didn't see the header.

Reply:
> we should clarify what the automatically granted MPRIS bus name is.


This is here now:

> as ``org.mpris.MediaPlayer2.$FLAPTAK_ID``